### PR TITLE
Add Readme and gitignore

### PR DESCRIPTION
**Pull request by bryanl**
- Adds readme with links to two blogs 
- Add gitignore to make sure bins aren't checked in

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+* [https://www.digitalocean.com/company/blog/taming-your-go-dependencies/](https://www.digitalocean.com/company/blog/taming-your-go-dependencies/)
+
+## Vendoring new code
+
+If you wish to add a third party dependency to the repo, put the code you
+vendor in `third_party/src/...`. Then rename the `.git` directory of that
+project to `.checkout_git`.
+
+```
+# do this for each new dependency
```
Comment (aybabtme) [inline README.md:13]: `go get ...`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# minithulhu
+
+An example for DigitalOcean's monorepo as described in:
+
+* [https://blog.gopheracademy.com/advent-2015/go-in-a-monorepo/](https://blog.gopheracademy.com/advent-2015/go-in-a-monorepo/)
+* [https://www.digitalocean.com/company/blog/taming-your-go-dependencies/](https://www.digitalocean.com/company/blog/taming-your-go-dependencies/)
+
+## Vendoring new code
+
+If you wish to add a third party dependency to the repo, put the code you
```
Comment (macb) [inline README.md]: Want to add the note that `go get github.com/my/dependency` would put the code in the correct place and fetch all deps of that dep?

**Comment by macb** [comment]
Ah @aybabtme beat me to it. aside from the small comment :+1:

File: README.md (hunk 2 above)
Comment (bryanl) [inline README.md]: will do

**Pull request merged**